src/app/Library/CrudPanel/Traits/Create.php

```diff
@@ -236,4 +237,42 @@ private function getRelationDataFromFormData($data)
 
         return $relationData;
     }
+
+    /**
+     * Transform the data before saving into the database. This is kind of an alias to Laravel Mutators but defined on Controler when defining the field.
+     * We basically call this "mutator" before saving the data into the database but after data is already validated.
+     *
+     * @param  array  $data  The form data.
+     * @return array The processed data.
+     */
+    protected function handleBeforeSavingCallback($data)
```

Comment:
I think this should be plural to be more accurate - `handleBeforeSavingCallbacks()`

Reply:
Agreed!